lighthouse-core/audits/autocomplete.js

```diff
@@ -18,24 +18,24 @@ const log = require('lighthouse-logger');
 
 const UIStrings = {
   /** Title of a Lighthouse audit that lets the user know if there are any missing or invalid autocomplete attributes on page inputs. This descriptive title is shown to users when all input attributes have a valid autocomplete attribute. */
-  title: 'Input elements use autocomplete',
+  title: '`<input>` elements use `autocomplete`',
```

Comment:
```suggestion
  title: '`<input>` elements correctly use `autocomplete`',
```

based on the failure title and description this feels more accurate?